R/label-utils.R

```diff
@@ -0,0 +1,145 @@
+#' @title Set Data Labels
```

Comment:
The amount of description, examples, etc. is minimal, but OK. 

Add more if you have time — but we can add documentation at a later release.